### PR TITLE
doc: change README.md

### PR DESCRIPTION
**Pull request by wendzhue**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Application (the showcase website) / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by wendzhue** [comment]
Preview README at https://github.com/wendzhue/ng-zorro-antd/tree/change-readme.

File: README-zh_CN.md [hunk 1]
```diff
@@ -39,12 +39,14 @@ Ant Design 的 Angular 实现，开发和服务于企业级后台产品。
 ## 🖥 支持环境
 
 - Angular `^6.0.0`
-- 现代浏览器，以及 Internet Explorer 10+ （使用 [polyfills](https://ant.design/docs/react/getting-started#Compatibility)）
+- 现代浏览器，以及 Internet Explorer 11+ （使用 [polyfills](https://ant.design/docs/react/getting-started#Compatibility)）
```
Comment (vthinkxie) [inline README-zh_CN.md]: link error

File: README-zh_CN.md [hunk 2]
```diff
@@ -52,28 +65,60 @@ $ ng add ng-zorro-antd
...
+## ⌨️ 开发
+
+```bash
+$ git clone git@github.com:NG-ZORRO/ng-zorro-antd.git
+$ cd ng-zorro-antd
+$ npm install
+$ npm run site:start
+```
+
+浏览器会自动打开 http://localhost:49152 。
```
Comment (vthinkxie) [inline README-zh_CN.md]: 端口号不固定

File: README-zh_CN.md [hunk 3]
```diff
@@ -52,28 +65,60 @@ $ ng add ng-zorro-antd
...
-> 如果你的公司和产品使用了 NG-ZORRO，欢迎到 [这里](https://github.com/NG-ZORRO/ng-zorro-antd/issues/1142) 留言。
+> `@angular/cli` 的用户不需要担心下面这项设置，但知道也挺有好处。
+
+然后在 `angular.json` 文件中引入样式。
 
-## 如何贡献
+```json
+{
+  "styles": [
+    "node_modules/ng-zorro-antd/src/ng-zorro-antd.min.css"
```
Comment (vthinkxie) [inline README-zh_CN.md]: 应该不需要src了?

File: README-zh_CN.md (hunk 2 above)
Comment (wendzhue) [reply README-zh_CN.md]: 为什么我每次打开都是这个端口, 换电脑也是... 奇怪...

Comment (vthinkxie) [reply README-zh_CN.md]: --port 0是找一个未被占用的端口，启动两次的话就不是这个端口了

**Pull request merged**